astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "Getting Started with FusionAuth: A Beginner's Guide"
+description: Learn how to set up FusionAuth, integrate authentication into your app, and implement advanced features like magic link logins in this developer-friendly guide.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/synedra/fusionauth-first-steps-launch/main/
```

Comment:
Can we change this now that the changes to https://github.com/FusionAuth/fusionauth-example-express-start-here/ are merged (or approved to be merged)?

Reply:
Done, committing in a moment.